test/e2e/specs/site-editor/template-registration.spec.js

```diff
@@ -101,7 +101,15 @@ test.describe( 'Block template registration', () => {
 	} ) => {
 		// Create a post.
 		await admin.visitAdminPage( '/post-new.php' );
```

Comment:
```suggestion
		await admin.createNewPost();
```

We should replace `visitAdminPage( '/post-new.php' )` with `createNewPost()`. It handles welcome modals internally. I think the test has two calls.

Reply:
Thank you! Done in 25c1962c6a0b68e459eb13d44d9e2cae39457bfe